help/faq/_posts/2017-10-24-faqs.md

```diff
@@ -66,6 +66,13 @@ The easiest method is to link your Classic account to your the new ADS account a
 ##### Q: I have some links to Classic pages on my website - do I need to update all of these to the new ADS links?
 Nope! Existing Classic URLs will continue to work, even after Classic is retired.
 
+##### Q: Where do I find the utility ***Get citation lists for selected articles*** and similar second order queries?
+In general, the function **Reviews** in the **Explore** menu will generate the same list of publications as selecting all records in the old ADS and clicking on the button *Get citation lists for selected articles*. The only difference is the fact that the new ADS does not offer the frequency: how many publications in the original set are cited by a publication in the result set. The results set in the old ADS is sorted by this frequency, so this is the only difference: a difference in sorting.
```

Comment:
Minor (but important) point: theoretically, the results of reviews() (or useful) should be sorted by the number of cites to the original set, when sorted by score. We just don't display the number anywhere. This function is currently broken, so I can't verify right now, but that's my memory of it.